plugin/src/main/resources/io/jenkins/plugins/analysis/core/portlets/IssuesChartPortlet/portlet.jelly

```diff
@@ -15,12 +15,10 @@
 
   <st:adjunct includes="io.jenkins.plugins.echarts"/>
 
-  <script>
-      {
-          const trendProxy = <st:bind value="${it}"/>;
-
-          echartsJenkinsApi.renderTrendChart('${it.id}-issues-chart', `false`, trendProxy);
-      }
-  </script>
+  <st:bind value="${it}" var="trendProxy"/>
+  <f:invisibleEntry>
```

Comment:
I think this could be changed to have inline style on the span hiding the element if someone is opposed to the current approach.